src/components/ui4/RowUi4.tsx

```diff
@@ -162,8 +162,7 @@ const getStyles = cacheStyles((theme: Theme) => ({
   },
   textHeader: {
     color: theme.secondaryText,
-    fontSize: theme.rem(0.75),
-    paddingRight: theme.rem(1)
```

Comment:
why did this change?

Reply:
This was a remnant style from when I used to have the right button laid out differently. Now it's causing too much spacing with long header text that shrinks too early. This was apparent in the FIO screen

Reply:
But now we have to verify nothing else broke. :(

Reply:
Please double, triple, quadruple verify that this still looks good everywhere you can find a RowUi4. And check on iOS and Android. 4.0 is about to launch today as all perf issues have been resolved.

Reply:
All of the weird unexpected bugs that came up from changes during our integration were due to various combinations of flex properties.

We're not messing with flex properties, this is just a constant padding that was _wrong._ Other callers that have long titles will also have this incorrect extra padding. 

This style is what's used when RowUi4 manages the title (and in this case the title in relation to the right button), which does not affect any custom child content or any other weird callers that manage the entire contents of the card. The title is separate from the children or body. This padding change is no more risky than adding font scaling back to the title.

Before:
![image](https://github.com/EdgeApp/edge-react-gui/assets/90650827/b55854f0-a1b1-4f33-8016-e5460a636c7d)

After:
![image](https://github.com/EdgeApp/edge-react-gui/assets/90650827/a014ff26-12fe-48cf-9dff-fa83340b323d)

Very minor change which is completely localized to the title.